Description: last minute tarball fix ...
Author: Andreas Pakulat <apaku@gmx.de>

--- a/kdevphpsupport.desktop
+++ b/kdevphpsupport.desktop
@@ -63,7 +63,7 @@ GenericName[zh_TW]=PHP 支援
 ServiceTypes=KDevelop/Plugin
 Icon=application-x-php
 X-KDE-Library=kdevphplanguagesupport
-X-KDevelop-Version=8
+X-KDevelop-Version=9
 X-KDevelop-Language=Php
 X-KDevelop-Args=PHP
 X-KDevelop-Interfaces=ILanguageSupport
